frontend/src/app/components/VariantTests.js

```diff
@@ -0,0 +1,41 @@
+
+import React from 'react';
+
+export class VariantTestCase extends React.Component {
+  render() {
+    return <span>{ this.props.children }</span>;
+  }
+}
+
+VariantTestCase.propTypes = {
+  value: React.PropTypes.string.isRequired
+};
+
+export class VariantTestDefault extends React.Component {
```

Comment:
Does `VariantTestDefault` need to specify the `propTypes`, like the `VariantTestCase` class above?